Resources/Prototypes/Catalog/VendingMachines/Inventories/pwrgame.yml

```
@@ -1,16 +1,17 @@
 - type: vendingMachineInventory
   id: PwrGameInventory
   startingInventory:
-    DrinkPwrGameCan: 4
-    DrinkEnergyDrinkCan: 4
-    DrinkGrapeCan: 2
-    DrinkRootBeerCan: 2
-    DrinkIcedTeaCan: 2
-    DrinkSolDryCan: 2
-    DrinkLemonLimeCan: 2
-    DrinkLemonLimeCranberryCan: 2
+    DrinkPwrGameCan: 5 #Starlight +1
+    DrinkEnergyDrinkCan: 5 #Starlight +1
+    DrinkGrapeCan: 3 #Starlight +1
+    DrinkRootBeerCan: 3 #Starlight +1
+    DrinkIcedTeaCan: 3 #Starlight +1
+    DrinkSolDryCan: 3 #Starlight +1
+    DrinkLemonLimeCan: 3 #Starlight +1
+    DrinkLemonLimeCranberryCan: 3 #Starlight +1
     DrinkFourteenLokoCan: 2
   contrabandInventory:
+    DrinkDrGibbCan: 4 #Starlight +1
     DrinkDrGibbCan: 3
```

```suggestion
    DrinkDrGibbCan: 4 #Starlight +1
```
that’s where the linter error comes from.